JetMETCorrections/Modules/plugins/QGLikelihoodDBWriter.cc

```diff
@@ -25,130 +25,158 @@ class  QGLikelihoodDBWriter : public edm::EDAnalyzer{
  public:
   QGLikelihoodDBWriter(const edm::ParameterSet&);
   virtual void beginJob() override;
-  virtual void analyze(const edm::Event&, const edm::EventSetup&) override {}
-  virtual void endJob() override {}
-  ~QGLikelihoodDBWriter() {}
+  virtual void analyze(const edm::Event&, const edm::EventSetup&) override {};
```

Comment:
Semicolons after method definitions on lies 28-30 are unnecessary, though harmless. If any other changes need to be made to this file, it would be good to also delete these semicolons to avoid confusion.